installer/README.md

```diff
@@ -18,34 +18,53 @@ Testing on other major cloud providers is in the roadmap.
 
 ## Prerequisites
 
-To use the Cloudflow GKE installer, you need to have the following packages installed on your local machine:
-
+### K8s CLI's
+You need the command line tool for your particular Kubernetes cluster:
 * [Google Cloud SDK](https://cloud.google.com/sdk/)
+* [Amazon CLI for EKS](https://eksctl.io/)
+
+### Utilities
 * [jq](https://stedolan.github.io/jq/)
 * [Helm v2](https://helm.sh/) *note: Cloudflow installer is currently not compatible with Helm v3*
 
 ## Installation Procedure
 
+### GKE
 To install Cloudflow on GKE it is a straightforward process:
 
 ```
 # create a GKE cluster with name <gke-cluster-name>
 $ ./create-cluster-gke.sh <gke-cluster-name>
 # install Cloudflow in that cluster
-$ ./install-gke.sh <gke-cluster-name>
+$ ./install.sh <gke-cluster-name> gke
 ```
 Replace above `<gke-cluster-name>` with the preferred name for
 your GKE cluster.
 
+### EKS
+Similarly, to install Cloudflow on EKS follow this process:
+
+```
+# create an EKS cluster with name <eks-cluster-name>
+$ ./create-cluster-eks.sh <eks-cluster-name> <aws-region>
```

Comment:
Ideally we want the same format as with the installer command. That means somthing similar to:
```./create-cluster.sh <cluster-name> <cloudf-provider>```.

Reply:
Hmm the only reason I didn't do it that way is because `/create-cluster-eks.sh` is going to become kinda difficult to maintain as soon as we start adding more cloud providers and more options for each Kubernetes cluster favor.

FWIW i started that way until i re3lize how messy the file was going to get

Reply:
Another option would be splitting the file and sourcing the right bash scripts per case. I dont like that either (bash becomes hard to maintain anyway), we can solve this in another PR.